armadillo/src/main/java/org/molgenis/armadillo/controller/ProfilesController.java

```diff
@@ -87,6 +87,60 @@ public List<ProfileResponse> profileList(Principal principal) {
     return auditor.audit(this::getProfiles, principal, LIST_PROFILES);
   }
 
+  @Operation(
+      summary = "List profiles",
+      description =
+          """
+                        If Docker management is enabled, this will also display each profile's Docker
+                        container status.
+                        """)
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "All profiles listed",
+            content =
+                @Content(
+                    array = @ArraySchema(schema = @Schema(implementation = ProfileResponse.class))))
+      })
+  @GetMapping(value = "status", produces = APPLICATION_JSON_VALUE)
+  @ResponseStatus(OK)
+  public List<Map<String, String>> getProfileStatus(Principal principal) {
+    return auditor.audit(this::getDockerProfileInformation, principal, LIST_PROFILES_STATUS);
+  }
+
+  private List<Map<String, String>> getDockerProfileInformation() {
+    List<ProfileResponse> profiles = runAsSystem(this::getProfiles);
+    List<Map<String, String>> result = new ArrayList<>();
```

Comment:
je voelde niet de behoefte er weer een hele klasse structuur van te maken. Dat is op zich prima, maar zou het zou kunnen helpen bij de autogegenereerde docs als je dat wel doet?